.taskcat.yml

```
@@ -32,6 +33,7 @@ tests:
       AvailabilityZones: $[taskcat_genaz_2]
       ClusterName: tcat-$[taskcat_random-string]-$[taskcat_current_region]
       ClusterRegion: US_WEST_1
+      DatabasePassword: override
```

Can we change this to something like below? Do you have any restrictions on the password values?
```suggestion
      DatabasePassword: $[taskcat_genpass_12]
```